linux/att/client.go

```diff
@@ -526,7 +528,7 @@ func (c *Client) sendReq(b []byte) (rsp []byte, err error) {
 			// returns an ErrReqNotSupp response, and continue to wait
 			// the response to our request.
 			errRsp := newErrorResponse(rsp[0], 0x0000, ble.ErrReqNotSupp)
-			c.Debugf("client: rsp % X", b)
+			c.Debugf("rsp: % X", b)
```

Comment:
probably should change this to match the format above with hex.EncodeToString() or alternatively just %x